sass/cd/cd-footer/_cd-footer.scss

```diff
@@ -24,22 +24,68 @@
 }
 
 .cd-footer__section {
+  margin: 0 auto;
   padding-bottom: 48px;
+
+  // When we know a section contains no content, we can set an empty variant
+  // to collapse it to zero-height which helps especially on mobile, without
+  // disrupting the layout on larger viewport widths.
+  &--empty {
+    padding-bottom: 0;
+  }
 }
 
-@include tablet {
-  .cd-footer__inner {
+//——————————————————————————————————————————————————————————————————————————————
+// Common Footer: Layout
+//
+// The entire layout for the Common Footer is defined here. Each section has a
+// BEM modifier assigned to it now, and the block-level class
+//——————————————————————————————————————————————————————————————————————————————
```

Comment:
All styles defining Common Footer's layout live here now. Each component file is now purely a component.. no layout stuff in them!